### PR TITLE
8.x-1.x #3025756 Create Drupal 8 version

### PR DESCRIPTION
**Pull request by guardiola86**
See: https://www.drupal.org/node/3025756

### CODE REVIEW

File: src/Controller/JavascriptManagerController.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * @file
+ * Contains \Drupal\js_manager\Controller\JavascriptManagerController.
+ */
+class JavascriptManagerController {
```
Comment (davidgrayston) [inline src/Controller/JavascriptManagerController.php]: We should see if we can use `Drupal\Core\Config\Entity\ConfigEntityListBuilder` or `Drupal\Core\Config\Entity\DraggableListBuilder`

File: src/Controller/JavascriptManagerListBuilder.php [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildRow(EntityInterface $entity) {
+    $row['label'] = $entity->label();
+    $row['type'] = $entity->getJsType();
+    $row['weight'] = $entity->getWeight();
+    $row['scope'] = $entity->getScope();
+    $row['exclude_admin'] = $entity->excludeAdmin() ? 'No' : 'Yes';
```
Comment (marcelovani) [inline src/Controller/JavascriptManagerListBuilder.php]: Use translation for the strings

File: src/Controller/JavascriptManagerListBuilder.php [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildHeader() {
+    $header['label'] = $this->t('Name');
+    $header['type'] = $this->t('Type');
+    $header['weight'] = $this->t('Weight');
+    $header['scope'] = $this->t('Scope');
+    $header['admin_page'] = $this->t('Include on admin pages');
```
Comment (marcelovani) [inline src/Controller/JavascriptManagerListBuilder.php]: For consistency with the variable `exclude_admin` we could use the same key here and the message could also say `Exclude admin pages`

Comment (marcelovani) [reply src/Controller/JavascriptManagerListBuilder.php]: Unless it makes more sense to keep same message as D7 branch

**Comment by davidgrayston** [comment]
Diff: c1151830d7bb4c4fc2c4a7777527a3d536925950...1f0f959ee87595d58a1cc5176f7e18c31132d415

- Converted to use Javascript config entity
- Invalidating cache tags to prevent the need for a full cache clear

**Pull request merged**